build.yaml

```diff
@@ -1,8 +1,8 @@
 nodejs:
   - "0.10"
   - "0.12"
-  - "iojs-3"
-  - "4.0"
+  - "4.3"
```

Comment:
Unfortunately we need to update the job creator code to allow those runtimes to be used (would be nice to make that dynamic, i'll look into that).  In near term i'll add those runtimes to job creator so they can be used here.